AuthenticatorShared/Core/Vault/Repositories/AuthenticatorItemRepository.swift

```diff
@@ -115,15 +115,14 @@ class DefaultAuthenticatorItemRepository {
         }
         .sorted { $0.name.localizedStandardCompare($1.name) == .orderedAscending }
 
-        let allItems = items.compactMap(ItemListItem.init)
+        let favorites = items.filter(\.favorite).compactMap(ItemListItem.init)
+        let nonFavorites = items.filter { !$0.favorite }.compactMap(ItemListItem.init)
 
         return [
-            ItemListSection(
-                id: "Everything",
-                items: allItems,
-                name: Localizations.all
-            ),
+            ItemListSection(id: "Favorites", items: favorites, name: Localizations.favorites),
+            ItemListSection(id: "Unorganized", items: nonFavorites, name: Localizations.myCodes),
```

Comment:
Do "Favorites" and "Unorganized" need to be localized?

Reply:
No, those are internal IDs, and never shown to the user